src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -258,5 +259,21 @@ export const GAME_OPTION_VALUES = [
                 value: MEASURE_TOOL_STYLE.ALL_ARCED
             }
         ]
+    },
+    {
+        name: GAME_OPTION_NAMES.CHAT_LOG_DURATION,
+        defaultValue: '3',
+        description: 'Chat Log Duration',
+        help: 'How long chat transmissions will remain visible before fading out',
+        type: 'number',
+        onChangeEventHandler: EVENT.CHAT_LOG_DURATION_CHANGE,
+        unit: 'sec',
+        validationHandler(newVal) {
```

Comment:
is `validationHandler` a new concept? I like it! There are some rooms for abstraction here, but I think it's probably too soon so we can leave it like this.

Reply:
Yes, new concept